whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/MntLowerAttributeValidationSpec.groovy

```diff
@@ -422,14 +423,15 @@ class MntLowerAttributeValidationSpec extends BaseQueryUpdateSpec {
         )
 
         then:
-        ack.errors
+        ack.failed
```

Comment:
modify inetnum (assigned pi and ripe maintained): **add mnt-lower** of lir mnt by lir